setup.py

```diff
@@ -61,6 +61,10 @@
         "scripts/runwvsnc2waves.py",
         "scripts/aqdturnaround.py",
         "scripts/exoturnaround.py",
+        "scripts/runwxtcsv2cdf.py",
```

Comment:
Thanks for adding in the wxt run files but they are a little out of scope of this PR. Just a thought for the future.

Reply:
Noted.